ipfx/stimulus.py

```diff
@@ -26,7 +26,10 @@ def has_tag(self, tag, tag_type=None):
 
 class StimulusOntology(object):
 
-    DEFAULT_STIMULUS_ONTOLOGY_FILE = os.path.join(os.path.dirname(__file__), 'defaults/stimulus_ontology.json')
+    DEFAULT_STIMULUS_ONTOLOGY_FILE = os.path.join(
+        os.path.dirname(__file__), 
+        'defaults/stimulus_ontology.json'
```

Comment:
Should this be `"defaults", "stimulus_ontology.json"` to remove the slash from a path name? I don't actually know how `os.path.join` handles that cross-platform.